packages/suite/src/components/suite/modals/index.tsx

```diff
@@ -56,7 +56,7 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 
 type Props = ReturnType<typeof mapStateToProps> &
     ReturnType<typeof mapDispatchToProps> & {
-        background?: boolean;
+        background?: boolean | string;
```

Comment:
weird combo

Reply:
otherwise undefined ?? true is true, need to be false, so passing an empty string